### ISSUE TITLE
Add a DisplayText field for AdminMenu nodes to allow specifying a custom display title

### ISSUE THREAD

**Issue by CrestApps** [OrchardCMS/OrchardCore#11650]
### Is your feature request related to a problem? Please describe.
When selecting ContentType to show it on the AdminMenu node, you can't change how the menu item reads. By default we show the display name of the content type. Typically, the content-type name is in singular form like Customer, Opportunity, Client.... However, on the menu, I would like to display them as Customers, Opportunities, Clients just because these links would lend the user to a list of content-items. so when you click on Customers you get a list of customer.

### Describe the solution you'd like
I think It would be nice to add an optional "DisplayText" option to the content items when one can specify what text should be shown on the menu. If no text specify, then we can continue to display the display name of the content type.

![image](https://user-images.githubusercontent.com/24724371/167018710-93448276-7633-41c4-bc4d-ff4179e0a870.png)

One may say "Can't you just change the display name of the content-type". We'll you could but then the button to add a content item won't make scene like "Add Customers" since you are adding a single customer at a time.

**Comment by hishamco** [comment]
Please have a look to TheBlog recipe, there's a `Blog` menu item where it lists a blog posts, you can change the link text to "Blog Posts" for instance

**Comment by CrestApps** [comment]
Yes this can be done from recipe by setting the DisplayText value. However, it can’t be done using the UI. I think adding this into the UI would be helpful.

**Comment by hishamco** [comment]
What if the user select all content types?!! It needs a little thinking to make the UI looks much better

**Comment by Skrypt** [comment]
Normally we should display the `Content-Type` DisplayName.

**Comment by ns8482e** [comment]
IMO OOB display name field is enough for UI friendly name.

For menus, There is currently way to customize menu, in navigation builder add Id and implement an template override for menu with an Id is enough for what you are trying to accomplish
